docs/update.md

```diff
@@ -0,0 +1,37 @@
+# Updating Groups
+
+Inago includes an `update` command, which allows updating
+a group on the cluster.
+
+__Note__: The `update` command only works on sliced groups.
```

Comment:
`only _currently_ works on sliced groups.`

I think it's important to make it clear that this is functionality we want in the future.